src/routes/ConsortiumManager/settings/data-export/hooks/useDataExportLogs/useDataExportLogs.js

```diff
@@ -22,17 +23,18 @@ const JOB_LOGS_QUERY_VALUE = [
   FILE_STATUSES.FAIL,
 ].join(' or ');
 
+export const DATA_EXPORT_LOGS_QUERY_KEY = 'data-export-logs';
+
 export const useDataExportLogs = (params = {}, options = {}) => {
   const {
     pagination = DEFAULT_PAGINATION,
     sorting = DEFAULT_SORTING,
     tenantId,
   } = params;
 
-  const [namespace] = useNamespace({ key: 'data-export-logs' });
+  const [namespace] = useNamespace({ key: DATA_EXPORT_LOGS_QUERY_KEY });
   const ky = useTenantKy({ tenantId });
-  const stripes = useStripes();
-  const settingsPerms = stripes.hasPerm('ui-data-export.settings.view') && !stripes.hasPerm('ui-data-export.view');
+  const { hasPerm } = useConsortiumManagerContext();
```

Comment:
Using the correct `hasPerm()` function is key to this PR's success. How do you feel about these two identically-named functions that take different arguments? Would it be valuable to add an alias like `hasTenantPerm()` to the tenant-specific function?